### PR TITLE
Update rubocop

### PR DESCRIPTION
**Pull request by whereismyjetpack**
updates rubocop for ruby 3 support.

### CODE REVIEW

**Comment by whereismyjetpack** [comment]
@ajkiessl -- this is ready to go. Let me know if there's anything I can do.

File: exe/niftany [hunk 1]
```diff
@@ -10,7 +10,7 @@ require 'scss_lint/cli'
 require 'colorize'
 
 def parse_opts(args)
-  options = OpenStruct.new
+  options = Struct.new
```
Comment (ajkiessl) [inline exe/niftany:13]: @whereismyjetpack Getting this error here when I try to run niftany:

`13:in `new': wrong number of arguments (given 0, expected 1+) (ArgumentError)`

Is OpenStruct deprecated?

Comment (ajkiessl) [reply exe/niftany:13]: It works with `OpenStruct.new`

**Review by ajkiessl — CHANGES_REQUESTED** [review]
Also, I'm getting a deprecation message for the `--auto-correct` flag.  It should be `--autocorrect`.  So these need to be changed: https://github.com/psu-libraries/niftany/blob/c46992e0294f80b90c302415ff1bdef28b7eee77/exe/niftany#L39
https://github.com/psu-libraries/niftany/blob/c46992e0294f80b90c302415ff1bdef28b7eee77/exe/niftany#L39

**Review by ajkiessl — APPROVED** [review]

**Pull request merged**